views/news/news-item.js

```diff
@@ -30,7 +30,9 @@ export default function NewsItemView({story: {content, title}}: {source: string,
         color: black;
         text-decoration: none;
       }
-
+      iframe {
+        max-width: 100%
```

Comment:
Please add a semicolon to the end of this line. CSS requires semicolons to separate declarations.